packages/strapi-design-system/src/TimePicker/TimePicker.tsx

```diff
@@ -7,15 +7,31 @@ import { ComboboxInput, ComboboxInputProps, Option } from '../Combobox/Combobox'
 import { useDesignSystem } from '../DesignSystemProvider';
 import { Field, FieldError, FieldHint, FieldLabel, FieldProps } from '../Field';
 import { Flex } from '../Flex';
+import { useControllableState } from '../hooks/useControllableState';
 import { useDateFormatter } from '../hooks/useDateFormatter';
 import { useId } from '../hooks/useId';
 
+const isPrintableCharacter = (str: string): boolean => {
+  return Boolean(str.match(/^[^a-zA-Z]*$/));
```

Comment:
I find that confusing: if I read the regex currently it says: match any string that starts with a non-alphabetical character, which contradicts the util name.

Reply:
> which contradicts the util name

does it? I suppose the regex value infers what a printable character is... we could rename it to be `isNotAlphabeticalCharacter`?